### DISCUSSION TITLE
VisiData Advent Calendar?

### DISCUSSION THREAD

**Discussion by frosencrantz** [saulpw/visidata#1221] [General]
What do people think about doing a VisiData Advent Calendar?    It is very late to ask, and might be something better left until 2022.  I'll plant the idea here. I do enjoy following along the latest in various technical blog Advent calendars or  solving puzzles ([Advent of Code](https://adventofcode.com/)).

I was thinking it would be a great way to help new/old users learn something new about VisiData.   The type of content I’ve seen are things like puzzles to solve, or articles that focus on a particular topic, or a about a useful module/plugin.  Here are some possible ideas for content:

- Articles about plugins
- Data cleaning/transforming/searching challenges
- Tip-of-the-day.
- Advanced topics.
- Links to useful existing videos/articles/tutorials.

This community has a few blog writers. 

-Felix

P.S. Despite suggesting the idea, I'm not currently volunteering to do this.  This would require effort from different humans.

**Comment by saulpw** [comment]
This is a good idea, @frosencrantz, and I've thought about this myself. I agree it would be best if it incorporated ideas from several different people.  The main thing it needs is someone to organize it, and I don't have time for that myself this year.  If someone else has energy in this direction I'd love to help make this happen though!

**Comment by frosencrantz** [comment]
I agree it would happen if there was an organizer.  I do not have the time this year, either.  If no one steps forward this year, maybe next year we ask for volunteers in October or the start of November.

The least effort one would be just daily tweets of existing pluigins, tutorials, blog entries, interviews, youtube videos.  This would mostly be a reruns for anyone following,  but might be interesting for new users.  I think there are easily 24 of those.  Even various hints buried in these discussions and issues.

**Comment by 0ceanlight** [comment]
That's a neat idea! I also had the thought that we see how far we get solving Advent Of Code puzzles using VisiData. Even if one can't solve all the challenges completely, using vd to reformat the input in a useful way could also be a fun challenge.

**Comment by frosencrantz** [comment]
It is almost the start of autumn, is there enough of a lead time we could get something started for this year?  I think the first message still explains what we could do.

I'm still not volunteering, but I think it would be a great community project. @saulpw or @anjakefala not sure if you have time.  Or if someone else would like to do the honors.

**Comment by frosencrantz** [comment]
November is almost upon us,  time is running out.   I wanted to wake this thread again.  @saulpw @anjakefala There are a few users who we might be able to call out  who use VisiData that also share content.  In addition to Anja and Saul,  there is  AJ, Andrea, Jeremy, Joseph (reagle), Julia, Kris (videos), likely a few others...

**Reply by saulpw** [reply]
Hi @frosencrantz, we haven't made this public yet, but your comment triggered a different project, that we've been working on for the past month or so: [Hanukkah of Data](https://chanukah.bluebird.sh/), which will be a series of thematically-related data puzzles released over the 8 days of Hanukkah this year.  It's not VisiData-specific, but of course VisiData can certainly be used to solve the puzzles!  We'll be making a more official announcement in a couple of weeks, but in the meantime you can consider this a privileged leak :)

**Reply by frosencrantz** [reply]
That looks brilliant!  Can't wait.  Thanks, @saulpw!

**Reply by anjakefala** [reply]
And we are live! =) https://hanukkah.bluebird.sh
